src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/Preview.js

```diff
@@ -216,6 +216,10 @@ class Preview extends React.Component<Props> {
         this.selectedDeviceOption = value;
     };
 
+    @action handleDateTimeChange = debounce((value: ?Date) => {
+        this.previewStore.setDateTime(value || new Date());
```

Comment:
do we need to debounce this because of the datepicker component we are using?

Reply:
If we don't do that, the preview rerenders on every change. That includes every click on the up and down button for the time picker, so if you click ten times the minute up button to travel 10 minutes, a request for rerendering the preview will be sent 10 times to the server, if we don't debounce.